joblib/__init__.py

```diff
@@ -58,7 +58,10 @@
    inputs and  outputs: Python functions. Joblib can save their
    computation to disk and rerun it only if necessary::
 
+
       >>> from joblib import Memory
+      >>> import shutil ; import os
+      >>> if os.path.exists('/tmp/joblib'): shutil.rmtree('/tmp/joblib/')
```

Comment:
I don't really like that it makes the docstring harder to read. Maybe it could be moved elsewhere.